src/common/quantile.cc

```diff
@@ -366,6 +366,39 @@ void AddCutPoint(typename SketchType::SummaryContainer const &summary, int max_b
   }
 }
 
+template <typename SketchType>
+void AddCutPointSecure(typename SketchType::SummaryContainer const &summary, int max_bin,
+                     HistogramCuts *cuts) {
+    // For secure vertical pipeline, we fill the cut values corresponding to empty columns
+    // with a vector of minimum value
+    const float mval = 1e-5f;
```

Comment:
where is this used? also, `1e-5` is not the minimum feature value, XGBoost can take all real number left and right to 0.

Reply:
removed